src/App.js

```diff
@@ -4,65 +4,85 @@ import Main from './Main.js';
 import Footer from './Footer.js';
 import data from './data.json';
 import Modal from 'react-bootstrap/Modal';
-import Button from 'react-bootstrap/Button'
-import { CardImg, ModalBody } from 'react-bootstrap';
+import Form from 'react-bootstrap/Form';
+import Container from 'react-bootstrap/Container';
+
 
 class App extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      selected: false,
+      zoomed: false,
       image: 'placeholder',
-      blurb: 'Lorem Ipsum'
+      blurb: 'Lorem Ipsum',
+      cardDisplayed: true,
+      hornsSelectedState: 'all',
     }
   }
 
-
   handleRenderModal = (imageUrl, description) => {
     this.setState({
-      selected: true,
+      zoomed: true,
       image: imageUrl,
       blurb: description,
+
     })
   }
 
   handleRemoveModal = () => {
     this.setState({
-      selected: false
+      zoomed: false
     })
   }
 
-  render() {
-    return (
-      <>
-        <Header />
-
-        <Main 
-        data={data} 
-        handleRenderModal={this.handleRenderModal}
-        />
+  handleSelectForm = e => {
+    e.preventDefault();
+    let hornsSelected = e.target.value;
+    this.setState({
+      hornsSelectedState: hornsSelected
+    })
+}
```

Comment:
- We know that we want to set the state to be whatever the user selects from the dropdown (if horns is 2, then 2, if horns is 3, then show beasts with 3).
- Perhaps we can work in using `filter()` into this method. Keep track of what you are storing state and what value horns will be 👍